tests/mintcb/README.md

```diff
@@ -0,0 +1,8 @@
+mintcb
+======
+
+This test shows how to create a minimal-TCB enclave that only links
+**liboecore** (thereby avoiding **liboeenclave** and its dependencies).
+
+Ordinarily, the **_start** entry point is linked from **liboeenclave**. This
+test, links this entry point defined in **liboestart**.
```

Comment:
I like the test but wondering if you can print the TCB size. We can then compare this with regular TCB enclave size (may need a similar test that links from liboeenclave).